integration/testdata/6.0/config/puma.rb

```diff
@@ -17,7 +17,7 @@
 environment ENV.fetch("RAILS_ENV") { "development" }
 
 # Specifies the `pidfile` that Puma will use.
-pidfile ENV.fetch("PIDFILE") { "tmp/pids/server.pid" }
+pidfile ENV.fetch("PIDFILE") { "/tmp/pumd.pid" }
```

Comment:
What is this change for?

Reply:
Ah. I didn't explain that the changes in this PR are meant to support stacks that have differing UIDs in their build/run images.

The primary change is to log to `stdout` instead of a file that we would need to set group-writable permissions for.

The specific change here is to resolve another file permissions issue that crops up in stacks with these different UIDs. The pidfile path specified in the original is relative to the working directory and would thus require permissions changes to work. Instead of ensuring those permissions changes would work, we should just recommend to users that they write these types of temporary files to a group-writable location like `/tmp`. I've made that change here in the integration tests so that we are following our own future advice.

Reply:
@ryanmoran thank you for this explanation, that makes sense!